diag_manager/fms_diag_yaml.F90

```diff
@@ -166,6 +165,7 @@ module fms_diag_yaml_mod
   character (len=:), private, allocatable :: var_outname !< Name of the variable as written to the file
   character (len=:), private, allocatable :: var_longname !< Overwrites the long name of the variable
   character (len=:), private, allocatable :: var_units !< Overwrites the units
+  real(kind=r4_kind), private             :: var_zbounds(2)  !< indices of the z axis limits (zbegin, zend)
```

Comment:
If these are used as indices, then this variable should be an integer.

Reply:
ah yeah, good catch. 
Originally we were going to use just indices, but the values are actually given as pressures so it has to be real. I forgot to update the comment ...

https://github.com/NOAA-GFDL/AM4/blob/main/run/diag_table#L509